subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/page_object/WebView.kt

```diff
@@ -12,7 +12,7 @@ import org.hamcrest.Matcher
 import org.hamcrest.Matchers.containsString
 import java.util.concurrent.TimeUnit
 
-open class WebView(private val webViewMatcher: Matcher<View>) : PageObject() {
+open class WebView(private val webViewMatcher: Matcher<View>) {
```

Comment:
The same as https://github.com/avito-tech/avito-android/pull/1230/files#r712745017